calico-cloud/reference/installation/_ia_api.mdx

```diff
@@ -0,0 +1,131 @@
+<p>Packages:</p>
+<ul>
+<li>
+<a href="#image-assurance.operator.tigera.io%2fv1">image-assurance.operator.tigera.io/v1</a>
+</li>
+</ul>
+<h2 id="image-assurance.operator.tigera.io/v1">image-assurance.operator.tigera.io/v1</h2>
+<p>API Schema definitions for configuring the installation of Calico and Calico Enterprise</p>
```

Comment:
Seems like this should be updated to be image assurance specific

Reply:
Uggg, that's from a comment in the ia project.